innsender/src/main/kotlin/no/nav/soknad/innsending/supervision/InnsenderMetrics.kt

```diff
@@ -101,4 +116,14 @@ class InnsenderMetrics(private val registry: CollectorRegistry) {
 	fun archivingFailedSet(number: Long) = archivingFailedGauge.set(number.toDouble())
 
 	fun updateJobLastSuccess(jobName: String) = jobLastSuccessGauge.labels(jobName).setToCurrentTime()
+
+	fun fileNumberOfPagesSet(pages: Long) = fileNumberOfPagesSummary.observe(pages.toDouble())
```

Comment:
```suggestion
	fun setFileNumberOfPages(pages: Long) = fileNumberOfPagesSummary.observe(pages.toDouble())
```

Gjelder resten av funksjonene også